tech-far-hub/src/components/multi-step.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+
+import { Alert, Card, CardBody } from "@trussworks/react-uswds";
+
+export const MultiStep = ({ children }: { children: React.ReactElement }) => {
+  const errorMessage = (
+    <Alert type="warning" headingLevel="h3" heading="Error in process list">
+      A MultiStep list must be an ordered list.
+      <a href="https://www.markdownguide.org/basic-syntax/#ordered-lists">See the markdown guide for examples.</a>
+    </Alert>
+  );
+
+  if (children.type !== "ol") {
+    return errorMessage;
+  }
+
+  const elementFilter = (item: React.ReactNode) => item && typeof item === "object" && "type" in item;
+  let items = children.props.children.filter(elementFilter);
+
+  if (
+    !items.every((item: React.ReactNode) => item && typeof item === "object" && "type" in item && item.type == "li")
+  ) {
+    return errorMessage;
+  }
+  const [isVisible, setIsVisible] = React.useState([true, ...Array(items.length).fill(false)]);
+  const handleClick = (index: number): void => {
+    setIsVisible(() => {
+      const newIsVisible = Array(isVisible.length).fill(false);
+      newIsVisible[index] = true;
+      return newIsVisible;
+    });
+  };
+
+  const headings: React.ReactNode[] = [];
+  const bodies: React.ReactNode[] = [];
+  items.forEach((li: React.ReactNode, index: number) => {
+    console.log("\nNode: ");
```

Comment:
Not sure if you want to keep the console.logs on lines, 37, 40-42?

Reply:
Haha goodness I don't! Thank for that catch!